sdk/feature-set/src/lib.rs

```diff
@@ -869,6 +869,10 @@ pub mod reenable_sbpf_v1_execution {
     solana_pubkey::declare_id!("TestFeature21111111111111111111111111111111");
 }
 
+pub mod enable_secp256r1_precompile {
+    solana_pubkey::declare_id!("GkVUbiefEqFzzLcArWgNG7r3BCs551UUjdH2hVE5ns3E");
```

Comment:
who has this key? @samkim-crypto ?

Reply:
Key is still held by me, will need to have samkim generate one and replace it 🫡